src/quickstart.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+set -euo pipefail
+
+OWNER=microshift-io
+REPO=microshift
+BOOTC_ARCHIVE=""
+
+LVM_DISK="/var/lib/microshift-okd/lvmdisk.image"
+VG_NAME="myvg1"
+
+TMPDIR="$(mktemp -d /tmp/microshift-okd-XXXXXX)"
+trap 'rm -rf ${TMPDIR} &>/dev/null' EXIT
+
+function download_bootc_image() {
+    local -r filter="$1"
+
+    pushd "${TMPDIR}" >/dev/null
+    local -r url=$(curl -s https://api.github.com/repos/$OWNER/$REPO/releases/latest \
+        | grep "browser_download_url" \
+        | cut -d '"' -f 4 \
+        | grep "${filter}")
+
+    if [ -z "${url}" ]; then
+        echo "ERROR: No packages found for '${filter}'"
+        return 1
+    fi
+
+    echo "Downloading '${url}'"
+    curl -L --progress-bar -O "${url}"
+    popd >/dev/null
+
+    BOOTC_ARCHIVE="${TMPDIR}/$(basename "${url}")"
+}
+
+function install_bootc_image() {
+    local -r image_file="$1"
+
+    echo "Installing '${image_file}'"
+    podman load -q -i "${image_file}"
+}
+
+function prepare_lvm_disk() {
+    local -r lvm_disk="$1"
+    local -r vg_name="$2"
+
+    if [ -f "${lvm_disk}" ]; then
+        echo "INFO: '${lvm_disk}' already exists. Reusing it."
```

Comment:
Should we clear it? This procedure unconditionally stands up new container, so I think clearing makes sense